R/lancelot.R

```diff
@@ -2165,6 +2234,9 @@ lancelot_check_data <- function(data) {
                 "pillar2_50_64_tot", "pillar2_65_79_tot", "pillar2_80_plus_tot",
                 "pillar2_under15_pos", "pillar2_15_24_pos", "pillar2_25_49_pos",
                 "pillar2_50_64_pos", "pillar2_65_79_pos", "pillar2_80_plus_pos",
+                "admissions_0_9", "admissions_10_19", "admissions_20_29",
+                "admissions_30_39", "admissions_40_49", "admissions_50_59",
+                "admissions_60_69", "admissions_70_79", "admissions_80_plus",
```

Comment:
I think it's better if the naming of these is consistent with the age-aggregated version ( e.g. `all_admission_0_9`)